### DISCUSSION TITLE
The CONFIRM and CANCEL buttons for deleting mods are swapped.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#15263] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said

![image](https://github.com/user-attachments/assets/94ac1bc5-a5d6-4fe0-9f6a-29e4a2504403)

![image](https://github.com/user-attachments/assets/16235805-b059-461a-b943-d19889c60020)

### Reproduction steps

1. create or copy a mod
2. delete it
3. cancle deleting

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Whoops, fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/a5ff524b1c3dd21b5feba69120ffe9c757dbe40a (commit in our private repo). Fix coming up in the next unstable build and the upcoming release.
